mod-orders-storage/examples/piece_collection.sample

```diff
@@ -5,6 +5,7 @@
         "id": "5e317dc2-deeb-4429-b2a1-91e5cd0fd5f7",
         "caption": "Tutorial Volume 5",
         "comment": "Special Edition",
+        "format": "Electronic",
```

Comment:
Please add a few more pieces here of different format and content.

Reply:
Done